examples/EthernetHTTPS/EthernetHTTPS.ino

```diff
@@ -12,19 +13,38 @@
  modified 9 Apr 2012
  by Noah Koontz, based on work by Adrian McEwen and Tom Igoe
 
+ Modified 16 Oct 2019 by gdsports625@gmail.com for STM32duino_STM32Ethernet
  */
 
   // NOTE: This example REQUIRES the EthernetLarge library.
   // You can get it here: https://github.com/OPEnSLab-OSU/EthernetLarge
 
+#if defined(ARDUINO_NUCLEO_F767ZI)
+extern "C" {
+  // This must exist to keep the linker happy but is never called.
+  int _gettimeofday( struct timeval *tv, void *tzvp )
```

Comment:
Does this have to do with the Ethernet library on the Nucleo, or is it with SSLClient?

Reply:
This is the result of an interaction between SSLClient and the STM32Ethernet library.
It is not needed when using the STM32Ethernet library without SSLClient. Something in SSLClient is triggering this but I was never able to track this the exact library.

Reply:
I think this was caused by bearssl automatically resolving the "time.h" header. This issue should be fixed as well.